setup.py

```diff
@@ -1,19 +1,29 @@
+#!/usr/bin/env python
 # Copyright (c) Aaron Gallagher <_@habnab.it>
 # See COPYING for details.
 
 from setuptools import setup
 
 
+def get_requirements():
+    """returns a list of all requirements"""
+    requirements = []
+    with open('requirements.txt', 'rb') as infile:
+        for line in (l.decode('utf8').strip() for l in infile):
+            if line and not line.startswith('#'):
+                requirements.append(line)
+
+    return requirements
```

Comment:
I'd rather switch off of reading `requirements.txt` at all: https://caremad.io/posts/2013/07/setup-vs-requirement/